docs/reference/commands/setup-passwords.asciidoc

```diff
@@ -3,6 +3,8 @@
 [[setup-passwords]]
 == elasticsearch-setup-passwords
 
+deprecated[8.0, "The `elasticsearch-setup-passwords` tool is deprecated and will be removed in a future release. To manually reset the password for the `elastic` user, use the <<reset-elastic-password,`elasticsearch-reset-elastic-password` tool>>. To change passwords for other users, use either {kib} or the {es} change passwords API."]
```

Comment:
Drive-by comment: We should also add a deprecation notice to the [8.0 breaking changes](https://www.elastic.co/guide/en/elasticsearch/reference/master/migrating-8.0.html).

There isn't a deprecations section yet, but we can use the [7.15 one](https://www.elastic.co/guide/en/elasticsearch/reference/7.15/migrating-7.15.html#deprecated-7.15) as a template.

@lockewritesdocs Do you mind assisting and ensuring this gets added?

We'll also want to remove the `7.16` label if the `elasticsearch-reset-elastic-password` tool won't be in 7.x.